lib/trino-hdfs/src/main/java/io/trino/hdfs/HdfsEnvironment.java

```diff
@@ -112,4 +123,15 @@ public void doAs(ConnectorIdentity identity, Runnable action)
     {
         hdfsAuthentication.doAs(identity, action);
     }
+
+    private static void stopFileSystemStatsThread()
+    {
+        try {
+            Field field = FileSystem.Statistics.class.getDeclaredField("STATS_DATA_CLEANER");
```

Comment:
No need to do this not, but can we make this accessible next time we repackage Hadoop?